### DISCUSSION TITLE
OnlineJudge v1.6.1

### DISCUSSION THREAD

**Discussion by mikucat0309** [QingdaoU/OnlineJudge#458] [Announcements]
文档部分内容已不符合现在版本，正在建立新版英文文档，小弟英文不太好还请不吝赐教
未来以文档为准，不会再更新 OnlineJudgeDeploy README

### 新部署

<https://opensource.qduoj.com/#/en/guide/quickstart>

### 升级

<https://opensource.qduoj.com/#/en/management/upgrade>

## 改动

- [Judge Server] 修复一处内存泄漏弱点 QingdaoU/JudgeServer#59（感谢 @helsonxiao）
- [API Server] 更新依赖套件版本
- [API Server] 修复档案权限问题，使用与 judge server 相同的 UID 与 GID

---

English docuemnt is work in progress~

### New deployment

<https://opensource.qduoj.com/#/en/guide/quickstart>

### Upgrade

<https://opensource.qduoj.com/#/en/management/upgrade>

## Changes

- [Judge Server] Fix a memory leak vunlerbility QingdaoU/JudgeServer#59 (thanks @helsonxiao)
- [API Server] Bump dependecy versions
- [API Server] Fix testcase permission, use same UID and GID in API server and judge server

**Comment by FleyX** [comment]
发现一个小问题，docker-compose漏了中间的“-”
![image](https://github.com/QingdaoU/OnlineJudge/assets/29558551/398d9486-a511-4e81-a37c-365bc8c6bf2b)

**Reply by mikucat0309** [reply]
大部分安装方式都会将 Docker compose plugin 配置成 Docker CLI plugin，就能以 Docker CLI 子命令使用，所以两种方式都可以，官方文件使用子命令风格，所以我也跟着用。
reference: https://docs.docker.com/compose/install/linux/#install-the-plugin-manually

**Comment by FleyX** [comment]
这种安装方式略麻烦，我一般是直接用包管理器安装docker-compose,这种情况不会配置为plugin

**Reply by mikucat0309** [reply]
真要比较的话，用包管理器安装 docker compose plugin 也是一行指令的事，一样麻烦
